pep257.py

```diff
@@ -778,6 +778,17 @@ def check_newline_after_last_paragraph(self, definition, docstring):
                                  'quotes on separate line')
 
     @check_for(Definition)
+    def check_whitespaces(self, definition, docstring):
+        """D210: White spaces around doctring should be trimmed."""
```

Comment:
Change the message to `"D210: No whitespaces allowed surrounding docstring text."`